cmake/IgnSanitizers.cmake

```diff
@@ -17,11 +17,16 @@
 
 include(CheckCXXSourceCompiles)
 
-set(IGN_SANITIZER ""
+set(GZ_SANITIZER ""  # Back-ported from gz-cmake3
```

Comment:
I don't think we need these back-ported comments. We'll backport all variables that we can and this feels a bit polluting.

On the other hand, I think we should add "deprecated" comments to the IGN ones, without warnings.

Reply:
comment about backport removed in https://github.com/gazebosim/gz-cmake/pull/294/commits/364158f9aa5550dc4d50f339ba20b3fa8b9a536f

> On the other hand, I think we should add "deprecated" comments to the IGN ones, without warnings.

https://github.com/gazebosim/gz-cmake/pull/294/commits/595fe5d710719132fae3078e4a20ecc0b517748f